src/config/dynamic-theme-fixes.config

```diff
@@ -14524,6 +14524,27 @@ CSS
 
 ================================
 
+smash.gg
+
+CSS
+.ModalContext__close {
+    border-bottom: none !important;
+}
+.scroller-sgg2hsxy.hasHinting-sggj0FSG.horizontal-sgg1mjhU .controller-sggb2Rt_ {
```

Comment:
All these `sgg2hsxy` should not be used as they are randomly generated and can change any time. Something along the lines of `div[class*="scroller"]` should be used. Also, a [good resource](https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors) about the attribute selectors.

Reply:
done! thanks for pointing it out.